app/src/components/Note.js

```diff
@@ -10,6 +10,11 @@ export default function Note({ title, titleColor, text, textColor, link, textLin
         {text ? (
           <p className="text">
             {text}{" "}
+            <ul>
+              {sessions.map((session) => (
```

Comment:
si il n'y a pas de `sessions`, ca crash.
```suggestion
              {(sessions || []).map((session) => (
```